alonzo/test/lib/Test/Cardano/Ledger/Alonzo/AlonzoEraGen.hs

```diff
@@ -271,6 +280,11 @@ bigStep, bigMem :: Word64
 bigStep = 999 -- 9999999990
 bigMem = 500 -- 50000000
 
+instance HasField "totExunits" (Tx (AlonzoEra c)) ExUnits where
```

Comment:
I guess we had to copy the `ValidatedTx era` instance since here we have a `Core.Tx era`? Is there a way to combine them... :thinking: